app/assets/stylesheets/common/base/emoji.scss

```diff
@@ -4,6 +4,11 @@ img.emoji {
   vertical-align: middle;
 }
 
+img.emoji.all-emoji {
```

Comment:
Needs to be changed to `.only-emoji` as well 😉

Reply:
weird it was working with this on my screen 🤔  probably thought I rebaked and didnt :+1: